### PR TITLE
Epics

### PR DESCRIPTION
**Pull request by denolia**
- Add type input to the form
- Add epics input to the form
- Display a parent on the task
- Show epics board

image

### CODE REVIEW

File: src/Items/Board/Board.tsx [hunk 1]
```diff
@@ -115,27 +118,29 @@ export function Board() {
...
-                  <Column key={columnId} column={column} tasks={column.taskIds} index={index} />
-                );
-              })}
-              {provided.placeholder}
-            </Stack>
-          </div>
-        )}
-      </Droppable>
-    </DragDropContext>
+    <Box sx={{ m: 3 }}>
```
Comment (demoth) [inline src/Items/Board/Board.tsx:121]: what is the purpose of the outer `Box`?

File: src/Items/Calendar/CalendarView.tsx [hunk 2]
```diff
@@ -4,16 +4,22 @@ import { LoginRequired } from "../../Auth/Login/helpers/LoginRequired";
 import { ItemDrawer } from "../Drawer/ItemDrawer";
 import { useItems } from "../state/ItemContext";
 import { ItemDrawerProvider } from "../state/ItemDrawerContext";
+import { ItemType, Status } from "../types";
 import { Calendar } from "./Calendar";
 
+const defaultItem = {
```
Comment (demoth) [inline src/Items/Calendar/CalendarView.tsx:10]: sounds like a template rather than concrete item

File: src/views/EpicBoardView.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+const defaultItem = {
+  type: ItemType.EPIC,
+};
+
+export function EpicBoardView() {
+  return (
+    <LoginRequired>
+      <ItemDrawerProvider defaultItem={defaultItem}>
+        <Board filter={(item) => item.type === ItemType.EPIC} />
```
Comment (demoth) [inline src/views/EpicBoardView.tsx:16]: for future - prevent closing epic if some of the epics task are not closed

Comment (demoth) [reply src/views/EpicBoardView.tsx:16]: I can check that on the backend though

**Review by demoth — APPROVED** [review]

File: src/Items/Board/Board.tsx (hunk 1 above)
Comment (denolia) [reply src/Items/Board/Board.tsx:121]: only to add margin, maybe to be removed :thinking:

File: src/Items/Calendar/CalendarView.tsx (hunk 2 above)
Comment (denolia) [reply src/Items/Calendar/CalendarView.tsx:10]: It's Partial<Item>, I should have added the type :+1:

File: src/views/EpicBoardView.tsx (hunk 3 above)
Comment (denolia) [reply src/views/EpicBoardView.tsx:16]: I was thinking about automated status calculation on the basis of child statuses (lowest status of the children)

**Pull request merged**